projects/canopy/src/lib/spacing/row-gap/row-gap.notes.ts

```diff
@@ -0,0 +1,41 @@
+export const notes = `
+This directive allows you to add the \`row-gap\` CSS property using Canopy spacing.
+The spacing variables are also available as CSS custom properties (CSS variables)
+which can be viewed in _**canopy/projects/canopy/src/styles/spacing.scss**_.
+
+This directive and accompanying CSS classes, work great with our
+<a href="./?path=/docs/directives-grid--grid">Canopy Grid</a> but will
+also work with all other HTML elements that support
+<a href="https://developer.mozilla.org/en-US/docs/Web/CSS/row-gap" target="_blank" rel='noopener noreferrer'>CSS row-gap</a>.
+
+~~~js
+@NgModule({
+  ...
+  imports: [ ..., LgRowGapModule ],
```

Comment:
As per other comment I think I would move the directive within the grid module instead of having it as a separate module. 

I would move the above notes to the grid docs too.

Reply:
I did initially have it in grid but according to the CSS specs `row-gap` isn't just a property for flexbox (i.e. Canopy grid); it is also applicable to multi-column layouts and CSS grid.

Plus it uses the spacing 'sm', etc. so the grid stylesheet would either have to duplicate this, or create a dependency on the spacing stylesheet (the `$spacing-list` variable).

Reply:
Ah got it, thanks for clarifying! I think what you've done is perfectly fine then 👍